infra/src/main/kotlin/fi/fta/geoviite/infra/tracklayout/TrackLayout.kt

```diff
@@ -194,11 +195,17 @@ data class LocationTrack(
     val ownerId: IntId<LocationTrackOwner>,
     @JsonIgnore val alignmentVersion: RowVersion<LayoutAlignment>? = null,
     @JsonIgnore override val draft: Draft<LocationTrack>? = null,
+    @JsonIgnore val segmentSwitchIds: List<IntId<TrackLayoutSwitch>> = listOf(),
```

Comment:
Raiteelle lisätty kannasta suoraan joinilla sen vaihdelinkit jotta on helpompaa validoida. Käyttöä lienee muuallakin. Jos näitä halutaan fronttiin, tuon jsonignoren voi poistaa, mutta en laittanut defaulttina.